### PR TITLE
Rename list archived workflows command for consistency

### PR DESCRIPTION
**Pull request by feedmeapples**
## What was changed

Renamed `tctl workflow listarchived` to `tctl workflow list-archived`

## Why?

consistency in naming with other commands

## Checklist

1. Closes 

2. How was this tested:

unit tests

3. Any docs updates needed?

### CODE REVIEW

**Review by bergundy — COMMENTED** [review]
Why does this need to be a separate command? Could it just be a flag?

**Comment by robholland** [comment]
Flag sounds good to me.

**Comment by yycptt** [comment]
I'll vote for separate commands as the query capability, syntax, (and probably latency, depending on the archiver implementation) are very different for archived visibility records compared to non-archived ones. On the server side, we also have two different APIs due to this reason.

File: cli/workflow.go [hunk 1]
```diff
@@ -66,7 +66,7 @@ func newWorkflowCommands() []*cli.Command {
 			},
 		},
 		{
-			Name:  "listarchived",
+			Name:  "list-archived",
```
Comment (yycptt) [inline cli/workflow.go:69]: Do we need to worry about the rename breaking users of this command?

**Comment by bergundy** [comment]
> I'll vote for separate commands as the query capability, syntax, (and probably latency, depending on the archiver implementation) are very different for archived visibility records compared to non-archived ones. On the server side, we also have two different APIs due to this reason.

Can you elaborate what the differences are?
Are there enough similarities to justify putting them in a single command?

@yycptt

**Comment by feedmeapples** [comment]
closing in favor of https://github.com/temporalio/tctl/pull/198

**Pull request closed without merging**